source/fundamentals/logging.txt

```diff
@@ -0,0 +1,370 @@
+         provide a :ref:`custom sink <golang-logging-custom>` or integrate a :ref:`third-party
+         logger <golang-logging-thirdparty>` for the driver's logs. If you don't set this
+         property, driver uses the standard logging library.
+     - | ``SetSink()``
+       | **Parameter**: ``LogSink``
+
+   * - | ``MaxDocumentLength``
+       | **Type**: ``uint``
+       | **Default**: ``1000``
+     - | The maximum length in bytes of a document that the driver logs. If the
+         underlying document is larger than this value, the driver
+         truncates it and appends ellipses to the partial log message.
+     - | ``SetMaxDocumentLength()``
+       | **Parameter**: ``uint``
+
+.. tip:: Write Logs to a Specific File
+   
+   By default, the standard logger emits messages to your console (``stderr``). You can
+   specify a logging destination by setting the ``MONGODB_LOG_PATH``
+   environment variable to a filepath or ``stdout``.
+
+.. _golang-logging-component-severity:
+
+Logging Components and Severity Levels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify which components the driver will log against by
+specifying the ``LogComponent`` type. The driver provides the following
+built-in specifications for ``LogComponent``:
+
+- ``LogComponentAll``: enables logging for all components.
+- ``LogComponentCommand``: enables command monitor logging.
+- ``LogComponentTopology``: enables topology logging.
+- ``LogComponentServerSelection``: enables server selection logging.
+- ``LogComponentConnection``: enables connection services logging.
+
+You can also specify the logging component by using the enumeration of
+the preceding specifications. The following code shows equivalent ways
+of enabling command monitoring:
+
+.. code-block:: go
+
+   // Using named value
+   comp := options.LogComponentCommand
+
+   // Using enumeration
+   comp := options.LogComponent(1)
+
+You can specify the log severity level by specifying the ``LogLevel``
+type. The driver provides the following
+built-in specifications for ``LogLevel``:
+
+- ``LevelOff``: suppresses logging.
+- ``LevelInfo``: enables logging of high-level informational messages.
+- ``LevelDebug``: enables logging of detailed debug messages.
+
+You can also specify the severity level by using the enumeration of
+the preceding specifications. The following code shows equivalent ways
+of enabling logging at the ``INFO`` level:
+
+.. code-block:: go
+
+   // Using named value
+   lvl := options.LogLevelInfo
+
+   // Using enumeration
+   lvl := options.LogLevel(1)
+
+.. important::
+
+   The {+driver-short+} currently supports only ``DEBUG`` level messages.
+
+To learn more about logging severity levels, see the Wikipedia entry on
+the :wikipedia:`Syslog standard for message logging <Syslog#Severity_level>`.
+
+Example
+~~~~~~~
+
+This example shows how to configure a basic logger that uses the standard
+logging library and prints log messages. The logger has the following
```

Comment:
Q: Is there specific code that prints log messages here? If not, and it's standard behavior for a logger, I'd drop it